bot/resources/tags/keyerror.md

```diff
@@ -0,0 +1,17 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary. \
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
+## __The `dict.get` method__
```

Comment:
Discord doesn’t support markdown titles sadly 
```suggestion

**The `dict.get` method**

```

Reply:
Ah, figured the title would be used in the `Embed`s title field